presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreen.kt

```diff
@@ -26,6 +35,9 @@ fun EventScreen(
     onEditDescriptionClicked: (description: String) -> Unit,
     modifier: Modifier = Modifier) {
 
+    val calendarStateTimeDate = rememberUseCaseState()
+    var isStartDateTime = false
```

Comment:
This is a side-effect (something that's not Compose related, but is inside of a composable context). Every single time `EventScreen` recomposes, this `isStartDateTime` will be false again.